aidants_connect/settings.py

```diff
@@ -396,6 +396,10 @@ def getenv_bool(env_var: str, default_value: Union[None, bool] = None) -> bool:
 )
 CSP_SCRIPT_SRC = (
     "'self'",
+    "'unsafe-eval'",
```

Comment:
J'aime pas trop ça, on en a besoin pour quoi ?

Reply:
J'aime pas trop non plus mais on en a besoin pour [`.apply(this, arguments)`](https://github.com/betagouv/Aidants_Connect/pull/399/files#diff-62bc8737f3403de42aa4ea5180164f79a36b70e52c9d8cd9aa6684a8ced6feb1R399). Je suis pas sûr de voir le lien, normalement c'est pour la fonction `eval` mais Firefox semble le voir, lui :thinking:

Reply:
Si c'est pour IE on n'en a pas besoin, apparemment il ignore script-src de toute façon : https://caniuse.com/mdn-http_headers_csp_content-security-policy_script-src

Reply:
Effectivement. Ça sert à rien. Et ça marche bien sur IE11 😊